onnxruntime/core/providers/cpu/nn/pool_base.h

```diff
@@ -99,10 +99,13 @@ class LpPool {
 };
 
 class PoolBase {
+ private:
+  static inline bool IsGlobalPooling(const std::string& op_name) {
+    return op_name == "GlobalAveragePool" || op_name == "GlobalMaxPool" || op_name == "GlobalLpPool";
```

Comment:
The 'inline' keyword isn't necessary here as it is implicitly inline

[https://en.cppreference.com/w/cpp/language/inline](https://en.cppreference.com/w/cpp/language/inline)